src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -29,72 +29,66 @@ limitations under the License.
     <attribute side="server" code="0x0000" define="CONTENT_LAUNCHER_ACCEPT_HEADER" type="ARRAY" entryType="CHAR_STRING" length="254" writable="false" optional="false">accept header list</attribute>
     <attribute side="server" code="0x0001" define="CONTENT_LAUNCHER_SUPPORTED_STREAMING_PROTOCOLS" type="BITMAP32" writable="true" optional="false">supported streaming protocols</attribute>
 
-    <command source="client" code="0x00" name="LaunchContent" optional="false">
+    <command source="client" code="0x00" name="LaunchContentRequest" response="LaunchResponse" optional="false">
       <description>Upon receipt, this SHALL launch the specified content with optional search criteria.</description>
       <arg name="autoPlay" type="BOOLEAN"/>
       <arg name="data" type="CHAR_STRING"/>
-      <arg name="search" type="ContentLaunchParamater" array="true"/>
+      <arg name="search" type="Paramater" array="true"/>
```

Comment:
The type of this thing is "ContentSearch" in the spec, no?  And this field is not a list....

Also these fields don't have field ids that match the spec.  Followup for that...

Reply:
Updated.